src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexNode.cs

```diff
@@ -586,7 +586,7 @@ internal int Type()
         }
 
 #if DEBUG
-        internal static String[] TypeStr = new String[] {
+        internal static string[] TypeStr = new string[] {
```

Comment:
Nit: readonly?  I realize it's debug-only code, but if it's possible, it doesn't hurt